var/spack/repos/builtin/packages/py-fenics-ufl/package.py

```diff
@@ -14,17 +14,17 @@ class PyFenicsUfl(PythonPackage):
     mathematical notation."""
 
     homepage = "https://fenicsproject.org/"
-    url = "https://github.com/FEniCS/ufl/archive/2019.1.0.tar.gz"
-    git = "https://github.com/FEniCS/ufl.git"
+    url = "https://bitbucket.org/fenics-project/ufl/downloads/ufl-2019.1.0.tar.gz"
+    git = "https://bitbucket.org/fenics-project/ufl.git"
```

Comment:
According to https://bitbucket.org/fenics-project/ufl/src/master/, the package has moved to GitHub, so the previous url/git should be used

Reply:
Like with the FIAT package the UFL repository on GitHub has no .tar.gz archives, however I suggest to use the tag flag instead. As requested I will raise a separate PR.